Sources/MapboxCoreNavigation/PassiveLocationDataSource.swift

```diff
@@ -102,15 +108,15 @@ open class PassiveLocationDataSource: NSObject {
     /**
      Sets electronic horizon options. Pass `nil` to reset to defaults.
      */
-    public func set(electronicHorizonOptions: ElectronicHorizonOptions?) {
-        try! navigator.setElectronicHorizonOptionsFor(electronicHorizonOptions)
+    public func set(electronicHorizonOptions: EHorizonOptions?) {
+        try! navigator.setElectronicHorizonOptionsFor(electronicHorizonOptions?.nativeOptions)
     }
 
     public var graphAccessor: GraphAccessor
 
-    public var roadObjectsStore: RoadObjectsStore {
-        return try! navigator.roadObjectStore()
-    }
+    public lazy var roadObjectsStore: RoadObjectsStore = {
+        return RoadObjectsStore(try! navigator.roadObjectStore())
```

Comment:
Is it safe to stash the store here, or should there be a weak backpointer to `MapboxCoreNavigation.Navigator`?